win32more/__init__.py

```diff
@@ -441,18 +489,49 @@ def __commit__(self):
         return self._prototype()
 
 
+class ConstantWithAnnotations:
+    def __init__(self, prototype, annotation, module):
+        self._prototype = prototype
+        # Maybe generator could do this for us, like func/struct parameters hints.
+        if annotation not in EXCLUDED_ATTRS and not annotation.startswith('win32more.'):
+            annotation = f"{module.__name__}.{annotation}"

```

Comment:
is it possible `generate.py` write full path annotations for constants?

Reply:
no problem.